front/app/containers/Admin/pages-menu/routes.tsx

```diff
@@ -26,6 +28,22 @@ export default () => ({
       path: '',
       element: <CustomNavbarSettingsComponent />,
     },
+    {
```

Comment:
There's actually an in between part in the path. We're at pages-menu/homepage/homepage-banner for example, because there will be other pages to go to. I think this part of the route is added in 1008.

Reply:
The breadcrumbs should also be outside this page's content, but that is fixed with changing the routes perhaps? We don't need the Pages & menu + sub header.

Reply:
Routes should be sorted out once we have all the actual components merged to the feature branch, I don't think it makes sense to fix them in disparate branches